sass/components/hamburger.scss

```diff
@@ -0,0 +1,105 @@
+%hamburger-line {
+  display: none;
+  content: "";
+  position: absolute;
+  background: $color-logo;
+  border-radius: 5px;
+  height: 2.4px;
+  width: 100%;
+  @include media-768 {
+    display: inline-block;
+  }
+  @include media-375 {
+    display: inline-block;
+  }
+  @include media-320 {
+    display: inline-block;
+  }
+}
+
+.hamburger {
+  display: none;
+  position: relative;
+  z-index: 100;
+  width: 25px;
+  height: 20px;
+  @include media-768 {
+    display: inline-block;
+    margin-left: 20px;
+  }
+  @include media-375 {
+    display: inline-block;
+  }
+  @include media-320 {
+    display: inline-block;
+  }
+  &::before {
+    @extend %hamburger-line;
+    top: 0;
+    left: 0;
+  }
+  &::after {
+    @extend %hamburger-line;
+    bottom: 0;
+    left: 0;
+  }
+}
+
+.hamburger-rotate {
+  transform: rotate(90deg);
+  @extend %transition; 
+}
+
+.hamburger__line {
+  @extend %hamburger-line;
+  top: calc(50% - 1.2px);
+  left: 0;
+}
+
+.navigation-hamburger > * {
+  background: $color-bg;
+  width: 278px;
+  height: 812px;
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  padding: 162px 0 0 71px;
+  top: -800px;
```

Comment:
How do you calculate this 800px? Maybe there is a more reasonable decision for that?

Reply:
DONE!